python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -173,7 +173,8 @@ def transform(self, y: cudf.Series) -> cudf.Series:
             .set_keys(self._cats.keys())
             .values()
         )
-        if -1 in encoded:
+
+        if encoded.isin([-1]).any():
```

Comment:
why not `(encoded == -1).any()` there's probably a good reason but curious...

Reply:
Not sure what the exact difference is, to be honest. What was here before was actually a bug in cuDF, which has been fixed, and broke cuML as a result. I suppose either one will work. The other recommendation was to use `encoded.values`, which would bring all values back to host in a numpy array. This just seemed like a reasonable alternative (even though a numpy array is still created on host of size `n`).

Reply:
It would be ideal if, rather than having a numpy array of bools or the vals, we could just say "get me the first occurrence of this value" or "does this value exist in the Series at all?" but the API is apparently changing too quickly for us to couple ourselves to the `find_first_value()` method.